ammonia_test_script.py

```diff
@@ -0,0 +1,40 @@
+#####################################################################
+# This script tests the ammonia fitter                              #
+# with fake data produced by ammonia_model.n_modelfunc              #
+# using the xarr from G031.947+00.076_nh3_11_Tastar.fits [for now]  #
+#####################################################################
+
+from pyspeckit.spectrum.models import ammonia
+from pyspeckit import Spectrum
+from pyspeckit.spectrum.units import SpectroscopicAxis
+from astropy import units as u
+import numpy as np
+import matplotlib.pyplot as plt
+
+parameter_error_amplitude = [5., 5., 5., 5., 5., 5.]
+params = [2.3,2,0.,0.84,0.2,0.43]
+
+sp = Spectrum('G031.947+00.076_nh3_11_Tastar.fits')
+xarr = SpectroscopicAxis(np.linspace(-250, 385, 8192), refX=23694500000.0*u.Hz)
+# adding very very tiny noise
+parameter_noise = [params[0] + abs(np.random.randn()/1000.) * parameter_error_amplitude[0], 
+                   params[1] + abs(np.random.randn()/1000.) * parameter_error_amplitude[1],
+                   params[2] + abs(np.random.randn()/1000.) * parameter_error_amplitude[2],
+                   params[3] + abs(np.random.randn()/1000.) * parameter_error_amplitude[3],
+                   params[4] + abs(np.random.randn()/1000.) * parameter_error_amplitude[4],
+                   params[5] + abs(np.random.randn()/1000.) * parameter_error_amplitude[5]]
```

Comment:
Also... I'm very confused - why are these each value + noise?  This might be better called `parameter_with_noise`.  And you should probably use a loop for something this symmetric.